internal/backend/post.go

```diff
@@ -79,5 +85,35 @@ func (d *delphisBackend) NotifySubscribersOfCreatedPost(ctx context.Context, pos
 }
 
 func (d *delphisBackend) GetPostsByDiscussionID(ctx context.Context, discussionID string) ([]*model.Post, error) {
-	return d.db.GetPostsByDiscussionID(ctx, discussionID)
+	iter := d.db.GetPostsByDiscussionIDIter(ctx, discussionID)
+	return d.iterToPosts(ctx, iter)
```

Comment:
function to keep functionality before we decide to completely go towards raw sql